src/jalv_gtk.c

```diff
@@ -1226,6 +1224,8 @@ jalv_open_ui(Jalv* jalv)
 	gtk_window_present(GTK_WINDOW(window));
 
 	gtk_main();
+	suil_instance_free(jalv->ui_instance);
+	jalv->ui_instance = NULL;
```

Comment:
(I don't think my explanation above is accurate; it's been a long time since I examined the internals of glib. Maybe it is: GObject types last until exit, while classes are reference counted and finalized when no more instances exist. Anyway, best to let the event loop exit before closing the UI DLL.)